lib/bundler/cli/gem.rb

```diff
@@ -125,6 +125,15 @@ def run
         templates.merge!("CODE_OF_CONDUCT.md.tt" => "CODE_OF_CONDUCT.md")
       end
 
+      if ask_and_set(:rubocop, "Do you want to add rubocop as a dependency for gems you generate?",
+        "RuboCop is a static code analyzer that has out-of-the-box rules for many" \
+        "of the guidelines in the community style guide." \
+        "For more information, see the RuboCop docs (https://rubocop.readthedocs.io/en/latest/)" \
+        "and the Ruby Style Guides (https://github.com/bbatsov/ruby-style-guide).")
```

Comment:
The URL of Ruby Style Guide is now below.
https://github.com/rubocop-hq/ruby-style-guide

Reply:
Should I add the URL in parentheses after "in the community style guide"?